features/0212-pickup/README.md

```diff
@@ -60,10 +60,37 @@ A request to have multiple waiting messages sent inside a `batch` message.
 {
     "@id": "123456781",
     "@type": "https://didcomm.org/messagepickup/1.0/batch-pickup",
-    "batch_size": 10
+    "batch_size": 10,
+    "~timing": {
+      "delay_milli": 3000
+    }
 }
 ```
 
+`batch_size` is count of messages that **recipient** wants to retrieve
+
+#### message timings 
+[feature-0032](https://github.com/Purik/aries-rfcs/tree/0212_pickup/features/0032-message-timing)
```

Comment:
Should be the Hyperledger organization repo referenced here, not a personal repo.